docs/ispc.rst

```diff
@@ -886,6 +886,15 @@ New Architecture Support:
   Vector Extension (RVV) ISA. The new ``rvv-x4`` target provides 4-wide
   vectorization for RISC-V processors with vector extensions.
 
+Languages Changes:
```

Comment:
Overall, it's not that nice to have language definition to rely on this kind of vaguely defined things. I.e. there might be a expression that evaluates to a constant, which makes it not that crisp clear defined.

But it's probably the best we can do here.